### PR TITLE
Add module validator v8

### PR DESCRIPTION
**Pull request by hurrinico**
based on https://github.com/OCA/server-tools/pull/84

### CODE REVIEW

**Comment by hurrinico** [comment]
Actually this PR is Work In Progress

**Comment by pedrobaeza** [comment]
Marked as WIP. Tell me when it's done to remove the label

**Comment by eLBati** [comment]
:+1: 
the WIP tag can be removed for me

File: base_field_validator/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Nicola Malcontenti <nicola.malcontenti@agilebg.com>
```
Comment (hbrunn) [inline base_field_validator/README.rst:51]: you might want to add yourself as contributor

**Comment by hbrunn** [comment]
@hurrinico please lift the code to v8 api. I also put some comments to https://github.com/OCA/server-tools/pull/84, they apply to this PR just the same

**Comment by lmignon** [comment]
@hurrinico Is this PR a copy of #84 for 8.0? IMO, you should cherry-pick the commits from #84 to keep the history and make the changes required for 8.0 in a new commit.

**Comment by eLBati** [comment]
Let's continue on https://github.com/OCA/server-tools/pull/246. Please close this

@lmignon I rebuilt the commit history based on 7.0 version

@hbrunn I partially ported the module to new API. It was partial because my mind exploded while trying to handle calls to `write` and `create` methods supporting both old and new API arguments

**Comment by lmignon** [comment]
@eLBati Thank you for the rebase!

**Comment by hurrinico** [comment]
This PR is moved to https://github.com/OCA/server-tools/pull/246

**Pull request closed without merging**